openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Alerts.spec.js

```diff
@@ -0,0 +1,258 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import {
+    interceptURL, toastNotification, uuid, verifyResponseStatusCode
+} from '../../common/common';
+
+import { DELETE_TERM } from '../../constants/constants';
+
+const alertForAllAssets = `Alert-ct-test-${uuid()}`;
+const description = 'This is alert description';
+
+const TEST_CASE = {
+  testCaseAlert: `TestCaseAlert-ct-test-${uuid()}`,
+  testCaseDescription: 'This is test case alert description',
+  dataAsset: 'Test Case',
+  filters: 'Test Results === Failed',
+};
+
+describe('Alerts page should work properly', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.get('[data-testid="appbar-item-settings"]')
+      .should('exist')
+      .and('be.visible')
+      .click();
+    interceptURL('GET', '/api/v1/alerts', 'alertsPage');
+    cy.get('[data-testid="global-setting-left-panel"]')
+      .contains('Alerts')
+      .scrollIntoView()
+      .should('be.visible')
+      .and('exist')
+      .click();
+    verifyResponseStatusCode('@alertsPage', 200);
+  });
+
+  it('Create new alert for all data assets', () => {
+    interceptURL('GET', '/api/v1/alerts/*', 'createAlert');
+    //Click on create alert button
+    cy.get('button').contains('Create alert').should('be.visible').click();
+    verifyResponseStatusCode('@createAlert', 200);
+    //Enter alert name
+    cy.get('#name').should('be.visible').type(alertForAllAssets);
+    //Enter description
+    cy.get('#description').should('be.visible').type(description);
+    //Click on all data assets
+    cy.get('[title="All Data Assets"]').should('be.visible').click();
+    cy.get('[title="All Data Assets"]').eq(1).click();
+    //Select filters
+    cy.get('button').contains('Add Filters').should('exist').click();
+    cy.get('#filteringRules_0_name').invoke('show').click();
+    //Select owner
+    cy.get('[title="Owner"]').should('be.visible').click();
+    cy.get('.ant-select-selection-overflow')
+      .should('be.visible')
+      .click()
+      .type('Engineering');
+    cy.get('[title="Engineering"]').should('be.visible').click();
+    cy.get('#description').should('be.visible').click();
```

Comment:
Why description in case of owner?

Reply:
Its to close the dropdown once the selection is done